src/group.cpp

```diff
@@ -54,9 +54,11 @@ void Group::updatePeer(int peerId, QString name)
     toxids[toxid] = name;
 
     Friend *f = FriendList::findFriend(id);
-
     if (f != nullptr && f->hasAlias())
     {
+        peers[peerId] = f->getDisplayedName();
+        toxids[toxid] = f->getDisplayedName();
+    } else {
```

Comment:
Can you please fix the coding style, according to https://github.com/tux3/qTox/wiki/Coding

Else this is fine for me.